cucumber.cjs

```diff
@@ -0,0 +1,10 @@
+const {config} = require('./tests/e2e/config.cjs')
+
+module.exports = {
+    default: {
+        import: ["./tests/e2e/**/*.ts"],
```

Comment:
With this esm project, we have to create a separate `package.json` file inside `tests/e2e` directory to make it work with commonjs.
See: https://github.com/saw-jan/cucumber-ts-cjs-mjs for references
```suggestion
        require: ["./tests/e2e/**/*.ts"],
```